orbstation/code/map/shuttles.dm

```diff
@@ -22,6 +22,12 @@
 	credit_cost = CARGO_CRATE_VALUE * 5
 	emag_only = TRUE
 	admin_notes = "this things tiny as fuck"
+/datum/map_template/shuttle/emergency/maintshuttle
+	suffix = "maintshuttle"
+	name = "Maintenance Tsars Pleasure Barge"
```

Comment:
```suggestion
	name = "Maintenance Tsar's Pleasure Barge"
```

Reply:
i wanted to do this, but i wasn't sure if the ' would break things? cause sometimes in game it just turns into like random symbols in runechat or the endscreen and i didn't wanna like. break it. but i will make this change shortly